scripts/hak/hakEnv.ts

```diff
@@ -17,17 +17,17 @@ limitations under the License.
 import path from "path";
 import os from "os";
 import nodePreGypVersioning from "node-pre-gyp/lib/util/versioning";
-import { getElectronVersion } from "app-builder-lib/out/electron/electronVersion";
+import { getElectronVersionFromInstalled } from "app-builder-lib/out/electron/electronVersion";
```

Comment:
Can you explain how `getElectronVersionFromInstalled` differs from `getElectronVersion`, and why we need to change it as part of the TS translation? I couldn't find any documentation for them,

Reply:
`getElectronVersion` evaluates the config and sees whether the electron version is overridden there, which requires evaluating `electron-builder.ts` which errors in the hak scripts like in https://github.com/element-hq/element-desktop/actions/runs/8480242660/job/23235652340?pr=1591

Reply:
It looks at the version we dictate via package.json/yarn.lock, which is what `getElectronVersion` fell back to if it did not find an electron version specified in the builder config file (so no change for us)